src/main/java/net/dv8tion/jda/core/requests/Requester.java

```diff
@@ -180,7 +180,10 @@ public void shutdown()
 
     private BaseRequest createRequest(Route.CompiledRoute route, String body)
     {
-        String url = DISCORD_API_PREFIX + route.getCompiledRoute();
+    	String url = route.getCompiledRoute();
+    	if(!url.startsWith("https://")){                      // Some Group DMs Operations don't work properly using the default API prefix
+    		url = DISCORD_API_PREFIX.concat(url);             // so I did this to solve the problem of messing with things I shouldn't
+    	}                                                     //               yee
```

Comment:
We don't implement routes that are not supported by the default api, if you have to change the api prefix to `ptb` it should not be added in the first place. However I doubt that this endpoint only exists for `ptb`